pkg/registry/generic/etcd/etcd_test.go

```diff
@@ -256,8 +259,12 @@ func TestEtcdCreate(t *testing.T) {
 		},
 	}
 
+	ctx := api.NewContext()
+
 	for name, item := range table {
 		fakeClient, registry := NewTestGenericEtcdRegistry(t)
+		path, _ := registry.KeyFunc(ctx, "foo")
+		path = etcdtest.AddPrefix(path)
```

Comment:
I think part of the point of this test is to verify that the path is correct-- so this should just use ```etcdtest.AddPrefix("pods/foo")```... make sense?